keanu-project/src/test/java/io/improbable/keanu/tensor/bool/SimpleBooleanTensorTest.java

```diff
@@ -36,17 +37,17 @@ public void setup() {
     }
 
     @Test
-    public void youCannotCreateARankZeroTensor() {
-        thrown.expect(IllegalArgumentException.class);
-        thrown.expectMessage("Tensors must have rank >=2 : []");
-        SimpleBooleanTensor.create(new boolean[] {}, new long[] {});
+    public void youCanCreateARankZeroTensor() {
+        BooleanTensor scalarTrue = SimpleBooleanTensor.create(new boolean[]{true}, new long[]{});
```

Comment:
probably worth showcasing how to use the scalar API for this: 

```
BooleanTensor scalarTrue = SimpleBooleanTensor.scalar(true);
```

Reply:
.scalar(...) is a rank 0 true but this test shows that the `create` method can be used to create a rank 0 via new long[]{}.